pavelib/bok_choy.py

```diff
@@ -130,3 +165,25 @@ def bokchoy_coverage():
     sh("coverage html --rcfile={}".format(coveragerc))
     sh("coverage xml --rcfile={}".format(coveragerc))
     sh("coverage report --rcfile={}".format(coveragerc))
+
+
+@task
+def bokchoy_coverage():
```

Comment:
In the logs, it looks like it's already called:

```
 ---> pavelib.bok_choy.bokchoy_coverage
```

This task was already here with this name, but reorganized a bit.  The diff rendering makes it look like a new task.